core/src/test/java/org/infinispan/expiration/impl/ExpirationFunctionalTest.java

```diff
@@ -125,6 +125,10 @@ public void testExpirationLifespanInExec() throws Exception {
    }
 
    public void testExpirationMaxIdleInExec() throws Exception {
+      // Clustered caches can return max idle entries to executeTask
+      if (cache.getCacheConfiguration().clustering().cacheMode().isClustered()) {
+         return;
```

Comment:
IMO if we know `executeTask()` doesn't trigger maxIdle expiration, we should assert that

Reply:
Sure.